src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TypeTodo } from '../../types';
+import { FILTERS } from '../../constants';
+
+export const Footer = ({
+  todos,
+  clearCompleted,
+  handleFilter,
+}) => {
+  const todosInProgress = todos.filter(todo => !todo.completed);
+
+  const onClickHandleFilter = (filter) => {
```

Comment:
```suggestion
  const handleFilter = (filter) => {
```

Reply:
I have a conflict - the same name variable is declared in the upper scope 😪